fides/values.yaml

```diff
@@ -36,6 +36,8 @@ fides:
         value: "false"
       - name: FIDES__REDIS__SSL_CERT_REQS # Accepted values include: none, optional and require.
         value: "none"
+      - name: FIDES__EXECUTION__USE_DSR_3_0
+        value: "true"
```

Comment:
This is now deprecated (it only uses DSR 3.0) as of `2.70.0`. That being said, it's not _wrong_ to add it, but it will only take effect for those on `<2.70.0`.